### PR TITLE
rename org to stolostron

### PR DESCRIPTION
**Pull request by marcolan018**
https://github.com/stolostron/backlog/issues/18733
Signed-off-by: Marco llan@redhat.com

### CODE REVIEW

**Comment by clyang82** [comment]
🥇 

/lgtm

File: operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-statefulset.yaml [hunk 1]
```diff
@@ -112,7 +112,7 @@ spec:
         - --skip-provider-button=true
         - --openshift-ca=/etc/pki/tls/cert.pem
         - --openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-        image: quay.io/open-cluster-management/origin-oauth-proxy:2.0.11-SNAPSHOT-2021-04-29-18-29-17
+        image: quay.io/stolostron/origin-oauth-proxy:2.0.11-SNAPSHOT-2021-04-29-18-29-17
```
Comment (clyang82) [inline operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-statefulset.yaml:115]: cannot find this image. I am not sure how does it pass our test. @marcolan018 Do you know why?

Comment (clyang82) [reply operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-statefulset.yaml:115]: oh. I got it. Are you disable ci/prow/e2e-kind for release-2.4?

Comment (clyang82) [reply operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-statefulset.yaml:115]: We do not enable test in KinD for release branch. Do you think we need to enable it in release branch?

Comment (marcolan018) [reply operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-statefulset.yaml:115]: oh， for this one. I think it's required to be reverted, to use the image in open-cluster-management.

**Pull request merged**